exec/cnex/nstring.c

```diff
@@ -221,6 +233,35 @@ BOOL string_startsWith(TString *self, TString *s)
     return TRUE;
 }
 
+BOOL string_endsWithNoCase(TString *self, TString *s)
+{
+    size_t i, si;
+
+    for (si = 0, i = self->length - s->length; i < self->length; si++, i++) {
+        if (tolower(self->data[i]) != tolower(s->data[si])) {
+            return FALSE;
+        }
+    }
+
+    return TRUE;
+}
+
+BOOL string_endsWith(TString *self, TString *s, BOOL bCaseSensetive)
```

Comment:
Spelling ("sensitive").

Actually this shouldn't be a bool parameter at all. You've already got a function called `string_endsWithNoCase`, so make this one do a case sensitive compare. The caller can call the other one if they want case insensitivity.

Reply:
Changed.